crates/editor/src/element.rs

```diff
@@ -1035,15 +1047,14 @@ impl EditorElement {
                                         &[TextRun {
                                             len,
                                             font,
-                                            color: self.style.background,
+                                            color: cx.theme().colors().editor_background,
```

Comment:
Here `editor_background` may give you a value like `hsla(0.,0.,0.,0.)` in some themes. I assume this will make the text vanish, so we should have some sort of check to ensure this is a color that is at least partially opaque.